MobileCenterDistribute/MobileCenterDistribute/MSDistributePrivate.h

```diff
@@ -43,6 +43,11 @@ static NSString *const kMSUpdateTokenKey = @"MSUpdateToken";
 @interface MSDistribute ()
 
 /**
+ * View controller presenting the `SFSafariViewController`.
+ */
+@property(nonatomic) UIViewController *safariHostingViewController;
```

Comment:
Any reason that you keep this as a property? Opening a browser is just one time thing, you can dispose it after it is done.